server/apps/immich/src/controllers/share.controller.ts

```diff
@@ -24,6 +25,12 @@ export class ShareController {
     return this.service.getMine(authUser);
   }
 
+  @Authenticated()
+  @Get('partner')
+  getPartners(@GetAuthUser() authUser: AuthUserDto): Promise<PartnerResponseDto[]> {
+    return this.service.getAllPartners(authUser);
+  }
+
```

Comment:
This is really tricky, because if it is below the next get request it'll never get hit. This is just asking for trouble IMO.

One solution would be to update the link urls to have a `link` prefix:
```
GET /share/link
GET /share/link/me
GET /share/link/:id
GET /share/partner
```

However, I'd probably prefer to just keep all the partner sharing in the `PartnerController` and all of the shared links in this controller, which might be better named `SharedLinkController`. If each controller handles CRUD operations for the entity it is easier to follow.

We could move this endpoint to the `PartnerController` and use `GET /partner?sent=1` or similar to get this list back. 

The actual implementation of this method is identical to the one in partner controller, it just uses a different `PartnerDirection`, so I think it might be better for the url to exist inside of that controller.

Reply:
Moved partner sharing endpoints to `PartnerController`. Also renamed `ShareController` to `SharedLinkController`